pkg/services/alerting/notifiers/telegram.go

```diff
@@ -239,18 +239,18 @@ func generateImageCaption(evalContext *alerting.EvalContext, ruleURL string, met
 
 	if len(ruleURL) > 0 {
 		urlLine := fmt.Sprintf("\nURL: %s", ruleURL)
-		message = appendIfPossible(message, urlLine, captionLengthLimit)
+		message = appendIfPossible(log, message, urlLine, captionLengthLimit)
 	}
 
 	if metrics != "" {
 		metricsLines := fmt.Sprintf("\n\nMetrics:%s", metrics)
-		message = appendIfPossible(message, metricsLines, captionLengthLimit)
+		message = appendIfPossible(log, message, metricsLines, captionLengthLimit)
 	}
 
 	return message
 }
 
-func appendIfPossible(message string, extra string, sizeLimit int) string {
+func appendIfPossible(log log.Logger, message string, extra string, sizeLimit int) string {
```

Comment:
Nit. Would name the logger something else than `log` since it overrides the log package name.